docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -151,21 +153,27 @@ To demonstrate this flexibility, consider this nonsensical quantile on `span.htt
 { name = "GET /:endpoint" } | quantile_over_time(span.http.status_code, .99, .9, .5)
 ```
 
-### The `compare` function
 
-This adds a new metrics function `compare` which is used to split the stream of spans into two groups: a selection and a baseline.
+
+## The `compare` function
+
+The `compare` function is used to split the stream of spans into two groups: a selection and a baseline.
 It returns time-series for all attributes found on the spans to highlight the differences between the two groups.
-This is a powerful function that is best understood by looking at example outputs below:
+This is a powerful function that is best understood by looking at example outputs below.
 
 The function is used like other metrics functions: when it's placed after any search query, and converts it into a metrics query:
```

Comment:
```suggestion
The function is used like other metrics functions: when it's placed after any trace query, it converts the query into a metrics query:
```

I thought we have 'trace queries' and 'metrics queries' so suggesting this replacement.